api/types/registry/registry.go

```diff
@@ -102,3 +105,13 @@ type SearchResults struct {
 	// Results is a slice containing the actual results for the search
 	Results []SearchResult `json:"results"`
 }
+
+// DistributionInspect describes the result obtained from contacting the
+// registry to retrieve image metadata
+type DistributionInspect struct {
+	// Digest is the content addressable digest for the image on the registry
+	Digest digest.Digest
```

Comment:
@stevvooe do you mean rename the object to `Descriptor` or to use the `Descriptor` struct in blobs?